guide/src/format/configuration/general.md

```diff
@@ -46,6 +46,9 @@ This is general information about your book.
   `src` directly under the root folder. But this is configurable with the `src`
   key in the configuration file.
 - **language:** The main language of the book, which is used as a language attribute `<html lang="en">` for example.
+  This is also used to derive the direction of text (RTL, LTR) within the book.
+- **text_direction**: The direction of text in the book: Left-to-right (LTR) or Right-to-left (RTL). Possible values: `ltr`, `rtl`.
+  When not specified, the correct text direction is derived from the book's `language` attribute.
```

Comment:
I would avoid the word "correct" here since I see `text_direction` as the "correct" value vs the derived value which is more like a guess:

```suggestion
  When not specified, the text direction is derived from the book's `language` attribute.
```